### PR TITLE
[SAGE-203] Button update

### PR DESCRIPTION
**Pull request by QuintonJason**
## Description

- [x] remove `raised` prop
- [x] update button styles to new sage
- [x] add neutral variation
- [x] verify spinner variation
- ~[ ] add spinner to react variation~ [SAGE-278](https://kajabi.atlassian.net/browse/SAGE-278)
- [x] update docs

### Breaking changes
- remove the `raised` prop from the Rails and React buttons

## Screenshots

|  Before  |  After  |
|--------|--------|
|buttonBefore|buttonAfter|

## Testing in `sage-lib`

Visit the Button page:
- [Rails](http://localhost:4000/pages/component/button)
- [React](http://localhost:4100/?path=/story/sage-button--standard)

## Testing in `kajabi-products`

1. (**HIGH**) Update styles for the Button component across the application.

## Related

Closes [SAGE-203](https://kajabi.atlassian.net/browse/SAGE-203)

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/components/_button.scss [hunk 1]
```diff
@@ -487,13 +489,22 @@ $-alert-colors: (
...
   }
+
+  &.sage-btn--subtle {
+    border: 0;
+  }
 }
 
 .sage-btn--subtle {
-  padding: 0;
+  // padding: 0;
```
Comment (monicawheeler) [inline packages/sage-assets/lib/stylesheets/components/_button.scss:499]: Is this a line that can be removed? Or should we have a comment there to indicate why it's commented out?

File: packages/sage-assets/lib/stylesheets/components/_button.scss [hunk 2]
```diff
@@ -502,68 +513,31 @@ $-alert-colors: (
       background-color: transparent;
 
       // TODO: refactor the subtle button hover state, after Sage 3.0 conversion
```
Comment (monicawheeler) [inline packages/sage-assets/lib/stylesheets/components/_button.scss:515]: For this TODO text, is this something that can be removed now?

File: packages/sage-assets/lib/stylesheets/core/mixins/_sage.scss [hunk 3]
```diff
@@ -122,6 +123,24 @@
...
+///
+/// Sets a Sage-style custom focus outline to an interactive element.
+/// Should be paired with `sage-focus-outline--update-color`
+///
+/// @param {length} $focus-width [4px] Thickness of the focus ring itself
+/// @param {length} $focus-shadow [#8ECAFB] Hex color to be used for the background-color
+/// @param {time} $focus-animation-speed [0.2s] How quickly the focus ring animates on activation
+///
+@mixin sage-focus-ring(
+  $focus-shadow: #8ECAFB,
```
Comment (monicawheeler) [inline packages/sage-assets/lib/stylesheets/core/mixins/_sage.scss:135]: Are we able to convert this hex value to `sage-color(primary, 400)`?

**Review by monicawheeler — CHANGES_REQUESTED** [review]
Great work here! I've added a couple small items

**Review by monicawheeler — APPROVED** [review]
💯 great work!

**Review by philschanely — APPROVED** [review]
Solid work here! 🙌

**Review by pixelflips — APPROVED** [review]
LGTM! 👍🏼  Great work.

Not a showstopper at all, but I notice the not-allowed cursor is not being applied to the disabled buttons anymore. This also happens on the prod version. Just adding a note here as a reminder to create a follow-up.

**Review by anechol — CHANGES_REQUESTED** [review]
Looking very good Quinton! One minor thing. The font weights for this font are a bit weird. They're listed as `medium` in Figma when `medium` should be `semibold`. If you see anything labeled as Book, that would be the correct `medium` weight.

**Comment by QuintonJason** [comment]
> Looking very good Quinton! One minor thing. The font weights for this font are a bit weird. They're listed as `medium` in Figma when `medium` should be `semibold`. If you see anything labeled as Book, that would be the correct `medium` weight.

🔬 💯

**Pull request merged**